suites/portal/GWASUIAppTest.js

```diff
@@ -259,8 +272,11 @@ Scenario('Unauthorized access to GWAS @GWASUI', async ({
     home.do.goToHomepage();
     await home.complete.login(users.auxAcct1);
 
-    gwas.do.goToGWASPage();
+    gwas.do.goToAnalysisPage();
+    gwas.do.unauthorizedUserSelectTeamProject();
 
-    I.seeElement(gwas.props.UnauthorizedSpinner);
-    I.dontSeeElement(gwas.props.cohortTableColumn);
-});
+    // gwas.do.goToGWASPage();
```

Comment:
should this be uncommented?

Reply:
So that was the old way of checking it. With addition of the team project message box, the user would not be able to go to GWAS app anymore as the message box doesn go away unless you click on submit button. The button is greyed out for unauthorized user. I have kept it for now. I will remove at final cleanup. Still need to test the Results app part of code